dipper/graph/RDFGraph.py

```diff
@@ -174,12 +174,13 @@ def _getnode(self, curie):
         else:
             iri = RDFGraph.curie_util.get_uri(curie)
             if iri is not None:
-                node = URIRef(RDFGraph.curie_util.get_uri(curie))
+                node = URIRef(iri)
                 # Bind prefix map to graph
                 prefix = curie.split(':')[0]
-                if prefix not in self.namespace_manager.namespaces():
+                if prefix not in \
```

Comment:
I'm fairly certain this is causing the performance hit when comparing this PR to our last jenkins dipper run.  Our prefix check previously always evaluated to false, so we were re-binding the (1-3) namespaces every time we added a triple.  With the 5.0 I'm guessing bind has gotten more expensive, and I probably should have been comparing two different rdflib versions.  Graph.add() is much faster in 5.0 so I didn't consider the version change having an effect.